README.md

```diff
@@ -16,6 +16,36 @@ For details about configuration and deployment of RBD and CephFS CSI plugins, se
 
 For example usage of RBD and CephFS CSI plugins, see examples in `examples/`.
 
+## Support Matrix
+
+### Ceph-CSI features and available versions
+
+|         Volume Type                       |        Features                             | CSI driver Version |
+|-------------------------------------------|---------------------------------------------|--------------------|
+| File mode, sharable or RWX Volume(CephFS) | Dynamically provision, de-provision volume  |       v0.3.0       |
+|                                           | Creating and deleting snapshot              |          -         |
+|                                           | Provision volumes from snapshot             |          -         |
```

Comment:
https://github.com/ceph/ceph-csi/issues/246 mentions lack of snaps for CephFS?

Reply:
Yes, and hence the version column is unfilled(not supported in any of the versions as of yet).

Reply:
Yeah taking snapshots is no longer experimental but creating a new volume from the snapshot is TBD.  Snapshots exist under a hidden .snap subdirectory of a volume/share where old versions of files and subdirectories can be retrieved by the volume/share's users, so they have a use outside CSI even without the clone-to-new-volume capability.  We're looking to add the clone-to-new-share capability in Manila backed by CephFS and in addition Robert Vasek is thinking about the general CephFS CSI case when Manila isn't involved.